stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```

Comment:
@hveficent Licensing are inconsistent inside module.

Reply:
All moved to AGPL